<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DN8 Line</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>드릴리밍</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>가공시간</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ProcTime">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>15</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>303</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>검사시간</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_CheckTime">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>로딩시간</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>제자리</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_DR_LoadingTime">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>15</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>45</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>측면</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Side_LoadingTime">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>40</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_SP1_LoadingTime">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>15</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>20</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>SP1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>45</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>SP2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_SP2_LoadingTime">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_Start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>컨베이어</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>55</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>검사완료</string>
   </property>
  </widget>
  <widget class="QLabel" name="numChecked">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>55</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="numConv">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가공중</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Hello">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Hello</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>4STP1</string>
   </property>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>가공시간</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ProcTime_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>15</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>506</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>로딩시간</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>드-설</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_DR_LoadingTime_3">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>15</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>45</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>측-설</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Side_LoadingTime_3">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>40</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_SP1_LoadingTime_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>15</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>20</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>드-버</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>45</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>측-버</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_SP2_LoadingTime_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가공대기</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가공중</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>완료대기</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>완료대기</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가공중</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_7">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가공대기</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_8">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가공중</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_9">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가공대기</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_10">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가공대기</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_11">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>완료대기</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>4STP2</string>
   </property>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>가공시간</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ProcTime_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>15</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>506</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>로딩시간</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>드-설</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_DR_LoadingTime_4">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>15</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>45</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>측-설</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Side_LoadingTime_4">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>40</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_SP1_LoadingTime_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>15</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>20</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>드-버</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>45</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>측-버</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_SP2_LoadingTime_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_12">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가공중</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_13">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>완료대기</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_14">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>280</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가공중</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_15">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가공대기</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_16">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가공대기</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_17">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>완료대기</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>SIDE</string>
   </property>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>가공시간</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ProcTime_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>15</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>231</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>로딩시간</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>SP1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_DR_LoadingTime_5">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>15</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>45</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>SP2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Side_LoadingTime_5">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>40</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_18">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>가공중</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnProc_19">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>완료대기</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
